CIPs/cip-0046.md

```diff
@@ -0,0 +1,128 @@
+
+A forum post must exist for at least one week prior to rough consensus being achieved. The on-chain proposal must be submitted and the corresponding on-chain proposal ID must be shared in the forum post for at least 4 days prior to achieving rough consensus. This gives community members the opportunity to independently verify the on-chain exchange proposal.
+
+The exchange proposer is expected to answer reasonable questions from the community related to the exchange proposal, motivations, and funds.
+
+Signers of the approver multisig are expected to be active in the forum discussions, must signal their intent to approve at least two days before submitting the approval on-chain, and must post in the forum discussion when approval occurs. Approver multisig signers are able to wait longer than required before approving an exchange proposal, particularly in situations where an exchange proposal is controversial and determining rough consensus is not straightforward.
+
+Below is a table showing the expected timeline:
+
+| **Minimum # of days prior to achieving rough consensus** | **Actor**                 | **Action**                                                      |
+|----------------------------------------------------------|---------------------------|-----------------------------------------------------------------|
+| 7                                                        | Exchanger                 | Forum post created.                                             |
+| 4                                                        | Exchanger                 | Exchange proposal submitted on-chain & shared.                  |
+| 2                                                        | Approver multisig signers | On-chain proposal reviewed and intent for approval is signaled. |
+| 0                                                        | Approver multisig signers | Approval is submitted on-chain & shared.                        |                      |
+
+### Vetoing an exchange proposal
+
+Exchange proposals that have been approved can be vetoed by the community via a governance proposal. This is most appropriate if the approver multisig has approved an exchange proposal that did not achieve rough consensus.
+
+Anyone can submit a governance proposal by creating a CGP and proposing it on-chain. Passing a governance proposal takes multiple days, so it's important to create a veto governance proposal as early as possible to ensure the governance proposal will execute prior to the exchange proposal's veto period ending.
+
+The CGP will need to include a transaction that calls the `cancelExchangeProposal` function with the correct on-chain proposal ID.
```

Comment:
We can make it explicit that this proposal must only have a `cancelExchangeProposal` cal for the Celo Governance Approver multisig to approve it.